src/algebra/category/Module/abelian.lean

```diff
@@ -75,6 +75,26 @@ instance : abelian (Module R) :=
   normal_mono_of_mono := λ X Y, normal_mono,
   normal_epi_of_epi := λ X Y, normal_epi }
 
+section reflects_limits
+
+/- We need to put this in this weird spot because we need to know that the category of modules
+    is balanced. -/
```

Comment:
Minor nitpick
```suggestion
/- We need to put this in this weird spot because we need to know that the category of modules
    is balanced. -/
    
```
That way it's more obvious that the comment is about the whole section, not just one instance.